docker-entrypoint.sh

```diff
@@ -62,30 +67,40 @@ if expr "$*" : 'java .*/start\.jar.*$' >/dev/null ; then
 		esac
 	done
 
-	if [ -f /jetty-start ] ; then
-		if [ $JETTY_BASE/start.d -nt /jetty-start ] ; then
+	if [ $(whoami) != "jetty" ]; then
+		cat >&2 <<- EOWARN
+			********************************************************************
+			WARNING: User is $(whoami)
+			         The user should be (re)set to 'jetty' in the Dockerfile
+			********************************************************************
+		EOWARN
+	fi
+
+	if [ -f $JETTY_START ] ; then
+		if [ $JETTY_BASE/start.d -nt $JETTY_START ] ; then
 			cat >&2 <<- 'EOWARN'
 			********************************************************************
 			WARNING: The $JETTY_BASE/start.d directory has been modified since
-			         the /jetty-start files was generated. Please either delete 
-			         the /jetty-start file or re-run /generate-jetty-start.sh 
+			         the $JETTY_START files was generated. Either delete 
```

Comment:
Since we're using `'EOWARN'` above, this variable won't be interpolated. It seems like you want this to be interpolated. It's probably fine for `$JETTY_BASE` to be interpolated above as well, otherwise the `$` should be escaped.

Reply:
I'm in two minds about expanding those variables, as they can make the error message verbose.  Reporting with the env var syntax can be more concise and comprehensible.

you call... I'm on the fence.

Reply:
I tried the expansion and it reads OK because the paths are short, so committed